database/seeders/DatabaseSeeder.php

```diff
@@ -2,21 +2,136 @@
 
 namespace Database\Seeders;
 
-// use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use A17\Twill\Http\Controllers\Admin\MediaLibraryController;
+use A17\Twill\Models\Media;
+use App\Models\Theme;
+use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Seeder;
+use Illuminate\Http\Request;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Str;
 
 class DatabaseSeeder extends Seeder
 {
+    private array $localeSources = [
+        'en' => 'http://journeymaker.artic.edu/sites/default/files/json/data-en.json',
+        'es' => 'http://journeymaker.artic.edu/sites/default/files/json/data-es.json',
+        'zh' => 'http://journeymaker.artic.edu/sites/default/files/json/data-zh-hans.json',
```

Comment:
As a rule we generally don't commit domain names to code. Move these URLs to `.env` and retrieve them through `config()`.

Reply:
Added a `journeymaker` config file and update seeder to pull from there.